apps/projects/app/components/Content/Settings.js

```diff
@@ -310,56 +317,63 @@ BountyArbiter.propTypes = {
   networkType: PropTypes.string.isRequired,
 }
 
-const BountyContractAddress = ({ bountyAllocator, networkType }) => (
+const BountyContractAddress = ({ bountyAllocator, networkType, layoutName }) => (
   <div>
     <Text.Block size="large" weight="bold">
       Bounty Contract Address
     </Text.Block>
     <Text.Block>
       This is the smart contract that is actually allocating bounties.
     </Text.Block>
-    <div style={{ display: 'flex' }}>
-      <Viewport>
-        {({ below }) => {
-          const shorten = below('small')
-          return (
-            <LocalIdentityBadge
-              networkType={networkType}
-              entity={bountyAllocator}
-              shorten={shorten}
-            />
-          )
-        }}
-      </Viewport>
+    <div css="display: flex">
+      <LocalIdentityBadge
+        networkType={networkType}
+        entity={bountyAllocator}
+        shorten={layoutName === 'small'}
+      />
     </div>
   </div>
 )
 
 BountyContractAddress.propTypes = {
   bountyAllocator: PropTypes.string.isRequired,
   networkType: PropTypes.string.isRequired,
+  layoutName: PropTypes.string.isRequired,
+}
+
+const StyledInputDropDown = ({ children }) => {
+  const theme = useTheme()
+
+  return (
+    <div css={`
```

Comment:
Too much css going here, I wonder if we could solve this more elegantly with StyledComponents, if we start abusing these `css` approach for everything it may result confusing where to use components and css, I think the purpose of styled components is to organize and separate the styles a bit from the layout structure, so unless for minor one-liner styles I think we should try to avoid using `css` and define a StyledComponent instead, or better think on how to make re-usable styled components that can fit in several places!! That it what I would call efficiency 👍

Reply:
Point taken. I've seen a lot of css going on in other apps, and at some point it stopped being clear where to draw the line between styled component and inline css. Anyway, I think it's now improved.